client/index.html

```diff
@@ -45,6 +45,16 @@
 
                 <!-- CONTENT -->
                 <div id="content" class="col-md-9 col-md-offset-3 col-lg-10 col-lg-offset-2">
+                    <div class="pci-project-banner d-flex justify-content-between p-3 position-fixed col-md-9 col-lg-10">
+                        <span data-translate="cloud_welcome" class="font-weight-bold oui-color-sapphire"></span>
+                        <a class="oui-link pci-project-link_light"
+                            data-track-on="click"
+                            data-track-name="cloud_go-to-public-cloud'"
+                            data-track-type="action"
+                            data-ng-href="{{:: CloudMainController.PUBLIC_CLOUD_URL}}">
+                            <span data-translate="cloud_welcome_public_cloud"></span>
```

Comment:
`translate` directive can't be applied to the parent element?

Reply:
https://github.com/ovh-ux/manager/pull/688#discussion_r286371995